Helper/Subscription.php

```diff
@@ -330,5 +330,52 @@ public function cancelSubscription($id, $rzp)
         $postUpdate->setCancelBy('customer');
 
         $postUpdate->save();
-    }        
+    }
+
+    public function pauseSubscription($id, $rzp)
+    {
+        //fetch and pause subscription
+        $subscriptionResponse = $rzp->subscription->fetch($id)->pause(["pause_at"=>"now"]);
+
+        //update record
+        $subscription = $this->_objectManagement->create('Razorpay\Subscription\Model\Subscriptions');
+        $postUpdate = $subscription->load($subscriptionResponse->id, 'subscription_id');
+
+        $postUpdate->setStatus('paused');
+        $postUpdate->setCancelBy('customer');
+
+        $postUpdate->save();
+    }
+
+    public function resumeSubscription($id, $rzp)
+    {
+        //fetch and resume subscription
+        $subscriptionResponse = $rzp->subscription->fetch($id)->resume(["resume_at"=>"now"]);
+
+        //update record
+        $subscription = $this->_objectManagement->create('Razorpay\Subscription\Model\Subscriptions');
+        $postUpdate = $subscription->load($subscriptionResponse->id, 'subscription_id');
+
+        $postUpdate->setStatus('active');
+        $postUpdate->setCancelBy('customer');
+
+        $postUpdate->save();
+    }
+
+    public function allSubscriptionInvoice($subscriptionId, $rzp){
```

Comment:
Change the function name to `fetchSubsciptionInvoice` and add a document comment explaining what this function does

Reply:
i have changed the function name `allSubscriptionInvoice` to `fetchSubsciptionInvoice `